### ISSUE TITLE
workspaces `dev-dependencies` don't work

### ISSUE THREAD

**Issue by wmmc88** [eupn/macrotest#95]
I have a workspace where one of the crates uses `macrotest`. In that crate, I also put a workspace dev-dependency on another crate for other tests via `test-sys.workspace = true`. This causes macrotest to fail to pass. Other non-macrotest tests pass fine, and removing the workspace `dev-dependency` causes `macrotest` tests to pass

output:
```
---- tests::macro_expansion stdout ----
Running 1 macro expansion tests
Expansion error:
error: failed to parse manifest at `D:\test-ws\target\tests\test-package\macrotest002\Cargo.toml`
Caused by:
  error inheriting `test-sys` from workspace root manifest's `workspace.dependencies.test-sys`
Caused by:
  `workspace.dependencies` was not defined
```

macrotest version: latest on main [c4151a5](https://github.com/eupn/macrotest/commit/c4151a5f9f545942f4971980b5d264ebcd0b1d11)

**Comment by taiki-e** [comment]
Duplicate of #74

**Comment by taiki-e** [comment]
Fixed in 1.0.11 (https://github.com/eupn/macrotest/pull/104)
